app/Misc/Router.php

```diff
@@ -12,12 +12,28 @@ class Router
      */
     public function __construct()
     {
-        $dispatcher = FastRoute\simpleDispatcher(function(FastRoute\RouteCollector $collector) {
-            $collector->addRoute(['GET'], '/logout', [new \App\Controllers\FrontController(), 'logout']);
-            $collector->addRoute(['GET', 'POST'], '/[{table}[/{action}[/{id}]]]', [new \App\Controllers\FrontController(), 'main']);
-        });
-
         $factory = new Abimo\Factory();
+        
+        $front_controller_class = $factory->config()->get('app','front_controller_class') ;
+        
+        $dispatcher = FastRoute\simpleDispatcher(
+          function(FastRoute\RouteCollector $collector) use ( $front_controller_class ) {
+            
+              $collector->addRoute(['GET'], '/logout', 
+                  function() use ($front_controller_class) { 
+                      return ( new $front_controller_class )->logout() ; 
```

Comment:
to answer your question, the config setting has to be something legal for use by "new", - so it must be either a string to an existing (or autoloadable) class, or an object itself. Otherwise, the instantiation will fail.